File: python/fib-recursive.py
number = 30
def fib(number):
    if number == 0:
        return 1
    elif number == 1:
        return 1
    elif number == 2:
        return 2
    elif number > 2:
        return fib(number - 1) + fib(number - 2)


print("Fibonacci Sequence from 0 to", number) ## For formatting
print("======================================")
for n in range(0,number):
    print(fib(n), end=', ')

File: golang/fib-recursive.go
package main

import "fmt"

func fib(n int) int {
	if n <= 1 {
		return n
	}
	return fib(n-1) + fib(n-2)
}

func main() {
  number := 10
  fmt.Println("Fibonacci Sequence from 0 to", number)
  fmt.Println("================================")
	for i := 0; i <= number; i++ {
		fmt.Println(fib(i))
	}
}
